tests/unit/lms/services/jwt_oauth2_token_test.py

```diff
@@ -22,8 +22,7 @@ def test_save_new_token(self, svc, lti_registration, scopes):
         token = svc.save_token(lti_registration, scopes, "ACCESS_TOKEN", 3600)
 
         assert token.access_token == "ACCESS_TOKEN"
-        assert token.received_at == datetime(2022, 4, 4, 0)
-        assert token.expires_at == token.received_at + timedelta(seconds=3600)
+        assert token.expires_at == datetime(2022, 4, 4) + timedelta(seconds=3600)
```

Comment:
This doesn't exist on the model but our service uses it so our test also check for that :facepalm: